flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
any reason to exclude `aarch64` ?

Reply:
I will add it, but i don't have `aarch64` to test it